lib/scout_apm/utils/sql_sanitizer.rb

```diff
@@ -57,7 +60,8 @@ def to_s_sqlite
         sql.gsub!(SQLITE_REMOVE_STRINGS, '?')
         sql.gsub!(SQLITE_REMOVE_INTEGERS, '?')
```

Comment:
If you want to get fancy, you can do this in a single regex pass by doing:

```
sql.gsub!(SQLITE_REMOVE_STRINGS.union(SQL_REMOVE_INTEGERS), '?')
```

No clue if its faster, but it'd be one less call to `gsub!`